src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,72 @@
+package com.elixr.training;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+                if(args.length!=2)
+                {
+                    System.out.println("enter the valid path");
+                    return;
+                }
+                String inputfile = args[0];
+                String wordsearch = args[1];
```

Comment:
Could you try for a better variable name here too?

Reply:
changed it to searchword